### PR TITLE
updating eslint

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers this updates the global deps, including eslint. this has a new jsx check to put a space after your jsx before the /> so i did --fix so eslint would fix that up. there's also an underscore var name check i had to turn off until we make the rpc export not export underscore names

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -2,7 +2,9 @@
 /desktop/dist/**
 /desktop/release/**
 /desktop/node_modules
+/desktop/shared
```
Comment (chrisnojima) [inline .eslintignore]: these are picked up by ./shared anyways

File: .eslintrc [hunk 2]
```diff
@@ -27,6 +27,8 @@
         "arrow-parens": [2, "as-needed"],
         "react/jsx-no-bind": [2, {"allowArrowFunctions": true}],
         "react/prop-types": [0],
-        "filenames/filenames": [2, "^[0-9a-z-.]+(\\.desktop|\\.native|\\.ios|\\.android)?$"]
+        "filenames/filenames": [2, "^[0-9a-z-.]+(\\.desktop|\\.native|\\.ios|\\.android)?$"],
+        "camelcase": [0],
```
Comment (chrisnojima) [inline .eslintrc]: too many issues for now

File: .eslintrc [hunk 3]
```diff
@@ -27,6 +27,8 @@
...
+        "no-duplicate-imports": [0]
```
Comment (chrisnojima) [inline .eslintrc]: plugin doesn't understand flow imports

File: desktop/renderer/index.js [hunk 4]
```diff
@@ -124,4 +124,4 @@ class Keybase extends Component {
   }
 }
 
-ReactDOM.render(<Keybase/>, document.getElementById('app'))
+ReactDOM.render(<Keybase />, document.getElementById('app'))
```
Comment (chrisnojima) [inline desktop/renderer/index.js]: you'll see a shitload of these

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: desktop/renderer/index.js (hunk 4 above)
Comment (MarcoPolo) [inline desktop/renderer/index.js]: Why prefer this?

Comment (chrisnojima) [inline desktop/renderer/index.js]: i don't really care. this comes from standard. i want it one way or the other, right now its a mix

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**